Cartfile

```diff
@@ -1,3 +1,3 @@
-github "watson-developer-cloud/Swift-SDK" ~> 1.4.0
-github "watson-developer-cloud/RestKit" ~> 2.0.0
+github "watson-developer-cloud/Swift-SDK" ~> 2.0.0
```

Comment:
There is a difference in version number of the swift-sdk dependency between `Cartfile` and `Cartfile.resolved`. Does this matter?